prow/cmd/deck/static/pr/pr.ts

```diff
@@ -489,6 +492,9 @@ function createTitleLabel(isMerge: boolean, jobStatus: VagueState, noQuery: bool
     } else if (branchConflict) {
         label.textContent = "Blocked from merging into target branch";
         label.classList.add("pending");
+    } else if (authorConflict) {
```

Comment:
nit: This should probably be prioritized over branch conflicts since a PR can switch branches, but not authors.

Reply:
I moved it above the `branchConflict`.